charts/celeborn/templates/prometheus-podmonitor.yaml

```diff
@@ -40,12 +40,15 @@ spec:
       - {{ .Release.Namespace }}
   selector:
     matchLabels:
+      app.kubernetes.io/name: {{ .Chart.Name }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
```

Comment:
while rolling updating, there are maybe two version(i.e. v1, v2) both existing. If you add "app.kubernetes.io/version" as a matchLabel, then app.kubernetes.io/version equals v2 and then v1's pods will not be monitered.

Reply:
Sound reasonable to me, i'd like to remove `version` as a match label.